static/css/partials/_theme.scss

```diff
@@ -1,14 +1,4 @@
-.inverse-color-scheme {
-  background: $dark-default;
-  color: $light-default;
-
-  a {
-    color: $link-light;
-  }
-}
-
-
-.default-color-scheme {
+:not(iframe) body {
```

Comment:
Change here is breaking extension iframe UI, it seems we are using default and inverse color scheme in extension @chenba @6a68 including you for feedback here.